working-groups.md

```diff
@@ -0,0 +1,168 @@
+# Tekton working group
+
+Most community activity is organized into _working groups_.
+
+Working groups follow the [contributing](./CONTRIBUTING.md) guidelines although
+each of these groups may operate a little differently depending on their needs
+and workflow.
+
+The working groups generate design docs which are kept in a
+[shared drive](https://drive.google.com/drive/folders/0AFOvPxM9MpebUk9PVA)
+and are available for anyone to read and comment on. The shared drive currently
+grants read access to
+[tekton-users@](https://groups.google.com/forum/#!forum/tekton-users) and edit
+and comment access to the
+[tekton-dev@](https://groups.google.com/forum/#!forum/tekton-dev) Google
+group.
+
+Additionally, all working groups should hold regular meetings, which should be
+added to the
+[shared tekton calendar](https://calendar.google.com/calendar?cid=Z29vZ2xlLmNvbV9kM292Y3ZvMXAzMjE5aDk4OTU3M3Y5OGZuc0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t)
+WG leads should have access to be able to create and update events on this
+calendar, and should invite tekton-dev@googlegroups.com to working group
+meetings.
+
+Recordings are made and posted in the minutes of each meeting.
+These docs and the recordings themselves are visible to [all members of our mailing list](#mailing-list).
+
+
+The current working groups are:
+
+- [General](#general)
+- [Triggers](#triggers)
+- [CLI](#cli)
+- [Documentation](#documentation)
+- [Productivity](#productivity)
+- [Beta APIs](#beta-apis)
```

Comment:
i think this is called "beta release" down below

Reply:
:sweat_smile: